### PR TITLE
Added views import command.

### PR DESCRIPTION
**Pull request by baizmandesign**
Hi there,

I've been in contact with Juan de Paco about adding export and import commands for views. This is my attempt to add the import portion. Can someone review the pull request and let me know if it needs any revision?

Thanks!

Saul

PS. Views import is a little different from Types import because there are `current_user_can( EDIT_VIEWS )` conditionals in the Views plugin source code, which means that to run the import command in wp cli, one has to specify a WordPress user who has the appropriate capabilities, otherwise the import will fail. Types has no corresponding permission check. (But perhaps it should?)

In other words,
```
$ wp --user=<admin> views import <file> —> works
$ wp views import <file> —> fails
```

### CODE REVIEW

**Comment by zaantar** [comment]
Thanks again for your contribution and for the observation about user capabilities. I think we should do the opposite, in fact, make it so that you don't require a certain capability while running this from the command line - that means you can already do whatever you want with the site, even give yourself (super)admin capabilities, or access the database directly. There's no point in checking user caps. I'll raise this as a Views issue (however won't have the highest urgency, as you can probably imagine).

As for your PR itself, it looks good except like with #2, we need to add documentation. But I'll handle that when merging, no worries.

**Comment by baizmandesign** [comment]
@zaantar Thanks for your feedback. Your recommendation about skipping the user permissions check makes more sense. I guess the Views code could use a conditional to check whether the user has capabilities or that the constant WP_CLI is true.

As mentioned in #2, regarding documentation, I will edit the appropriate Markdown file and submit a new PR with documentation for both import and export commands.

Thanks again!

**Comment by zaantar** [comment]
Merged manually. 

Note to self: Created an issue views-4079 to address the issue with Views checking for user caps.
